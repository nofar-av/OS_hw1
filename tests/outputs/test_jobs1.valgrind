==27407== Memcheck, a memory error detector
==27407== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27407== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27407== Command: /home/omer/OS_wet1/smash
==27407== Parent PID: 27403
==27407== 
==27407== 
==27407== FILE DESCRIPTORS: 4 open at exit.
==27407== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_jobs1.valgrind
==27407==    <inherited from parent>
==27407== 
==27407== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_jobs1.err
==27407==    <inherited from parent>
==27407== 
==27407== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_jobs1.out
==27407==    <inherited from parent>
==27407== 
==27407== Open file descriptor 0:
==27407==    <inherited from parent>
==27407== 
==27407== 
==27407== HEAP SUMMARY:
==27407==     in use at exit: 0 bytes in 0 blocks
==27407==   total heap usage: 41 allocs, 41 frees, 82,940 bytes allocated
==27407== 
==27407== All heap blocks were freed -- no leaks are possible
==27407== 
==27407== For counts of detected and suppressed errors, rerun with: -v
==27407== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
