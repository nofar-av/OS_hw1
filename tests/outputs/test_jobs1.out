smash> smash> smash> smash> smash> smash> [1] sleep 100& : 28716 0 secs
smash> smash: sending SIGKILL signal to 1 jobs:
28716: sleep 100&
